manifests/kubernetes/deployment.yaml

```diff
@@ -81,6 +73,7 @@ spec:
         - 0.0.0.0:9102
         - -enable-gpu=true
         - -enable-cgroup-id=true
+	- -allow-running-inside-vm=true
```

Comment:
allow here to make CI happen

follow up, will update CI script to patch the manifest during start of test